config/application.yml.default

```diff
@@ -141,6 +141,15 @@ lexisnexis_trueid_liveness_nocropping_workflow: customers.gsa.trueid.workflow
 lexisnexis_trueid_noliveness_cropping_workflow: customers.gsa.trueid.workflow
 lexisnexis_trueid_noliveness_nocropping_workflow: customers.gsa.trueid.workflow
 ###################################################################
+# LexisNexis ThreatMetrix ##########################################
+lexisnexis_threatmetrix_base_url: https://www.example.com
+lexisnexis_threatmetrix_request_mode: testing
+lexisnexis_threatmetrix_account_id: test_account
+lexisnexis_threatmetrix_username: test_username
+lexisnexis_threatmetrix_password: test_password
+lexisnexis_threatmetrix_instant_verify_timeout: 1.0
+lexisnexis_threatmetrix_instant_verify_workflow: customers.gsa.instant.verify.workflow
```

Comment:
sorry one more thing, threatmetrix is one product, instant verify is another, we're going to be using them separetly, so we should make that clearer:
```suggestion
lexisnexis_threatmetrix_timeout: 1.0
lexisnexis_threatmetrix_workflow: customers.gsa.instant.verify.workflow
```

Reply:
Should the sample value also reflect that? i.e. `customers.gsa.threat.metrix.workflow` instead of `customers.gsa.instant.verify.workflow`

Reply:
In hindsight, yes that might be nice to update to a more realistic placeholder, but since it's a sample we should be replacing it anyways and I'm not super worried about it